Tests/CacheAdvanceTests/LorumIpsumMessages.swift

```diff
@@ -0,0 +1,502 @@
+//  Created by Dan Federman on 4/23/20.
+//  Copyright © 2020 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS"BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+struct LorumIpsum {
```

Comment:
enum? https://github.com/airbnb/swift#namespace-using-enums

Reply:
yeah I can adopt that part of our style guide here 🙂